packages/components/src/components/es-popover/es-popover.tsx

```diff
@@ -297,24 +299,23 @@ export class Popover {
 
         await popper.loaded();
         await this.positionPopper();
+
+        this.autoUpdateCleanup = autoUpdate(
+            this.attachTo ?? this.getParentNode()!,
+            this.popperShadow!,
+            () => this.positionPopper(),
+        );
+
```

Comment:
I think this could be kept in `attachDocumentListeners` (and the cleanup in `detachDocumentListeners`), just to keep it all in one place.